release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
+* You can filter Helm charts based on their certification level and see all the Helm charts in the *Developer Catalog*.
+* The Add page now features new options to create applications and associated services and deploy them on {product-title}. These include *Getting started resources*, *Creating applications using samples*, *Build with guided documentation*, *Explore new developer features* to name a few. Users can optionally hide specific cards on the Add page.
+* You can use workspaces in pipelines when creating via the pipeline builder or adding a trigger to a pipeline so that tasks can be used to support workspaces in pipelines.
+* You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications.
+* You can create multiple event source types in {product-title} that can be connected to sinks. Functions deployed as a Knative service on a {product-title} cluster, can be connected as a sink to Knative Eventing components.
+* You can use the `finally` task in the pipeline to execute in parallel. The `finally` task can guaranteed to be executed in parallel after all the pipeline tasks have completed regardless of the status of the tasks.
+* You can use the code assistance in add task form for the task parameter values. The pipeline parameter and the correct syntax for referencing that particular pipeline parameter would be referenced in the respective text field.
+* You can run a `Task` only when certain conditions are met, it is possible to guard task execution using the `when` field. The when field allows you to list a series of references to `when` expressions.
```

Comment:
`Run a `Task` after you meet the following conditions: <list conditions>. Use the `when` field to configure your tasks executions and to list a series of references to `when` expressions.`

I think you need to state what conditions must be met before a user can run a `Task`. Also, I was not sure what `guard task execution` meant, so I changed it to `configure tasks executions` does this make sense in this context? 

I also think the last bit can be removed. I don't know what the value is to listing a series of references to `when` expressions. So consider removing it so users simply focus on running the tasks and and configuring the tasks executions.

Reply:
It definitely makes sense now. Thanks!